CppCoreGuidelines.md

```diff
@@ -8060,6 +8061,22 @@ Typically, such programs leads to a painful effort to remove data races.
 
 **Enforcement**: Some is possible, do at least something.
 
+<a name="Rconc-voidf"></a>
+### CP.3: Consider void futures for one-shot event communication
```

Comment:
please move the html anchor inside the header as has been done for all the headers on the master branch.